streams/src/main/java/org/apache/kafka/streams/kstream/internals/KGroupedStreamImpl.java

```diff
@@ -105,10 +137,23 @@ public KGroupedStreamImpl(final KStreamBuilder topology,
         Objects.requireNonNull(aggregator, "aggregator can't be null");
         Objects.requireNonNull(windows, "windows can't be null");
         Objects.requireNonNull(storeName, "storeName can't be null");
+        return aggregate(initializer, aggregator, aggValueSerde, windowedStore(aggValueSerde, windows, storeName));
```

Comment:
This should be:
`return aggregate(initializer, aggregator, windows, aggValueSerde, windowedStore(aggValueSerde, windows, storeName))` - it is probably one of the reasons you have test failures